motr/m0crate/tests/test1_io.yaml

```diff
@@ -37,6 +37,7 @@ LOG_LEVEL: 4  # err(0), warn(1), info(2), trace(3), debug(4)
 WORKLOAD_SPEC:               # Workload specification section
    WORKLOAD:                 # First Workload
       WORKLOAD_TYPE: 1       # Index(0), IO(1)
+      POOL_FID: <0x6f00000000000001:0x2f> # The fid of the pool
```

Comment:
Each cluster configuration might have different fid for the default pool. I'd suggest to comment out this parameter so that the default pool could be used. When user needs to choose the pool, he should uncomment it and put the correct fid.

Reply:
Every deployment has different Pool ID.
This should be replaced when users use  this yaml configure to test.